fast-logger/System/Log/FastLogger.hs

```diff
@@ -259,6 +278,19 @@ newTimedFastLogger tgetter typ = case typ of
                 pushLogStr lgrset (f t)
                 when (cnt <= 0) $ tryRotate lgrset fspec ref mvar
         return (logger, rmLoggerSet lgrset)
+    dailyRotateLoggerInit fspec bsize = do
+        lgrset <- newFileLoggerSet bsize $ daily_log_file fspec
+        cache <- newTimeCache "%Y-%m-%dT%H:%M:%S"
```

Comment:
The format is hard-coded. Can we extend `DailyFileLogSpec` to specify a format?

Reply:
In principal absolutely. The problem is the current implementation (unlike the first) only supports daily rotation. This way I need to be able to figure out when a new day begins, which I cannot in general on a user supplied time format. Another way would be to use yet another time cache internally and one for the user specified format.